routes/team.js

```diff
@@ -28,4 +29,15 @@ team.post("/data", function(req, res) {
 	});
 });
 
-module.exports = team;
\ No newline at end of file
+team.post("/delete", authHelper.isAdmin, function(req, res) {
```

Comment:
Rather than posting to a delete endpoint you could delete to a user endpoint, its more inline with REST practice